content/guides/getstarted/full-stack-solana-development.md

```diff
@@ -1085,7 +1085,7 @@ Throw this command into your terminal to install all the `wallet-adapter` stuff
 we need:
 
 ```shell
-yarn add react @solana/web3.js \
+yarn add react @solana/web3.js@1 \
   @solana/wallet-adapter-base @solana/wallet-adapter-react \
```

Comment:
do we need to pin any of the wallet adapter packages? either in these install command or in their respective packages?

Reply:
I hope not, but probably.